test/kubectl-asset.integ.snapshot/lambda-layer-kubectl-integ-stack.template.json

```diff
@@ -1,380 +1,412 @@
 {
-  "Resources": {
-    "KubectlLayer600207B5": {
-      "Type": "AWS::Lambda::LayerVersion",
-      "Properties": {
-        "Content": {
-          "S3Bucket": {
-            "Fn::Sub": "cdk-hnb659fds-assets-${AWS::AccountId}-${AWS::Region}"
-          },
-          "S3Key": "c6964dbf0c556ec82ce09622e99ad6f6d4e488cdaac0ef9e8492e078ec61ffed.zip"
-        },
-        "Description": "/opt/kubectl/kubectl and /opt/helm/helm"
+ "Resources": {
+  "KubectlLayer600207B5": {
+   "Type": "AWS::Lambda::LayerVersion",
+   "Properties": {
+    "Content": {
+     "S3Bucket": {
+      "Fn::Sub": "cdk-hnb659fds-assets-${AWS::AccountId}-${AWS::Region}"
+     },
+     "S3Key": "c6964dbf0c556ec82ce09622e99ad6f6d4e488cdaac0ef9e8492e078ec61ffed.zip"
```

Comment:
Okay so, a bunch of things are changing in the integration tests, but the important hash remains the same 👍

Just commenting to make my review transparent.